tests/test_submission/test_integration.py

```diff
@@ -44,6 +44,29 @@ def teardown_method(self):
     def compare(self, a, b):
         return abs(a - b) <= 0.0001
 
+    def test_competition_field(self, tmpdir):
+        working_dir = str(tmpdir.mkdir('sub'))
+        config_dir = str(os.path.join(os.path.dirname(__file__), 'configs/'))
+        run_evaluation(config_dir, working_dir, 33, TestIntegration.database, models=['alexnet'],
+                       benchmarks=['dicarlo.MajajHong2015.IT-pls'])
+        model = Model.get()
+        assert model.competition == "cosyne2022"
+
+    def test_competition_field_none(self, tmpdir):
+        working_dir = str(tmpdir.mkdir('sub'))
+        config_dir = str(os.path.join(os.path.dirname(__file__), 'configs/'))
+        submission = Submission.create(id=33, submitter=1, timestamp=datetime.now(),
```

Comment:
isn't this submission changed to have the `competition` field set? https://github.com/brain-score/brain-score/pull/314/files#diff-5bb7cee62807d0b6b68c098d0ca440824cac9f167b7681d48f3055d009da48a8R91